sci-physics/tauola/tauola-1.1.8-r1.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit fortran-2
+
+MY_PN="TAUOLA"
+MY_P=${MY_PN}.${PV}
+
+DESCRIPTION="Tau decay Monte Carlo generator"
+HOMEPAGE="http://tauolapp.web.cern.ch/"
+SRC_URI="https://tauolapp.web.cern.ch/resources/${MY_P}/${MY_P}.tar.gz"
+S=${WORKDIR}/${MY_PN}
```

Comment:
```suggestion
S="${WORKDIR}/${MY_PN}"
```

Reply:
I am surprised pkgcheck did not complain, since it usually complains about unquoted `EPREFIX` and `FILESDIR`.

Reply:
That's because bash variable expansion is not really a problem when doing variable assignment. I like to quote stuff like WORKDIR/S/FILESDIR anyway because 1) aesthetic reasons and 2) I'm less likely to make a mistake if I simply quote it always.